### PR TITLE
Blog Post: 6th Wave of Nostr Grants

### PR DESCRIPTION
**Pull request by Arvin21M**
**Blog Preview:**
- [/blog/nostr-grants-august-2024](https://os-website-git-nostr-grants-august-2024-open-sats.vercel.app/blog/nostr-grants-august-2024)

**Social Preview:**
- image

### CODE REVIEW

**Comment by Arvin21M** [comment]
@dergigi social header/preview image ready. I have an alternative one to this ready if necessary to change.

**Comment by dergigi** [comment]
Love it! 🌊 💜

File: data/blog/nostr-grants-august-2024.mdx [hunk 1]
```diff
@@ -0,0 +1,199 @@
+---
+title: "Sixth Wave of Nostr Grants"
+date: '2024-08-08'
+tags: ['OpenSats', 'grants', 'nostr', 'wave']
+draft: false
+authors: ['arvin', 'dergigi']
+images: ['/static/images/blog/TODO']
```
Comment (dergigi) [inline data/blog/nostr-grants-august-2024.mdx:7]: Make sure to set the image here once you upload it to the `/blog` folder, like this:

```suggestion
images: ['/static/images/blog/45-nostr-grants-august-2024.jpg']
```

**Comment by Arvin21M** [comment]
@OpenSats/volunteers Team, would some one please have a look at this announcement for a sanity check? 🙏

File: data/blog/nostr-grants-august-2024.mdx [hunk 2]
```diff
@@ -0,0 +1,199 @@
+---
+title: "Sixth Wave of Nostr Grants"
+date: '2024-08-08'
```
Comment (dergigi) [inline data/blog/nostr-grants-august-2024.mdx:3]:
```suggestion
date: '2024-08-07'
```

**Review by dergigi — COMMENTED** [review]
Date should be today if it goes out today 📆

**Review by dergigi — APPROVED** [review]
Sanity checked: reads well, all links point to the right targets, metadata is correct, licenses are correct; in short: should be good to go!

Looks good to me ✅

**Pull request merged**